applications/vessel-history/src/features/home/Home.module.css

```diff
@@ -86,6 +91,16 @@
   padding: var(--space-S);
   border-top: var(--border);
 }
-.bottomActions > button {
-  flex-grow: 1;
+
+.bottomActionsBtn {
+  flex: 1;
+}
+
+@media only screen and (min-width: 1000px) {
```

Comment:
Careful with this, this media query will be applied also in the tablet screen (1280 × 800)

Reply:
Yes, I chose that number to make the changes appear when tablets are used on landscape mode.

Reply:
Yes, I chose that number so that this changes appear on the tablet when used in landscape mode. Is it a problem?